unittests/platformagnostictests.py

```diff
@@ -121,3 +121,10 @@ def test_update_wrapdb(self):
                     '''))
             subprocess.check_call(self.wrap_command + ['update-db'], cwd=testdir)
             self.init(testdir, workdir=testdir)
+
+    def test_none_backend(self):
+        testdir = os.path.join(self.python_test_dir, '7 install path')
+        self.init(testdir, extra_args=['--backend=none'], override_envvars={'NINJA': 'absolutely false command'})
+        self.assertPathDoesNotExist(os.path.join(self.builddir, 'build.ninja'))
+        out = self._run(self.meson_command + ['install', f'--destdir={self.installdir}'], workdir=self.builddir)
+        self.assertNotIn('Only ninja backend is supported to rebuild the project before installation.', out)
```

Comment:
I should verify that stuff gets installed too. Ideally also check that some tests can be run.

Reply:
Done and done.